site/content/en/docs/v0.4/reference/api.md

```
@@ -41,6 +41,25 @@ _Appears in:_
 
+#### BackupSnapshot
+
+
+
+BackupSnapshot describes the artefact produced by a successful
+EtcdBackup.
+
+
+
+_Appears in:_
+- [EtcdBackupStatus](#etcdbackupstatus)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)